### ISSUE TITLE
doesn't work

### ISSUE THREAD

**Issue by spyhunter99** [mil-oss/walkitalkie#1]
looks like most of the project is incomplete. it records and plays back audio but doesn't transmit

**Comment by obscurerichard** [comment]
Almost all the work done on this was completed in a single afternoon. @cpatrick did the heavy lifting on the record and playback of audio, I think nobody got to the transmit / receive part. Perhaps you'd like to give it a crack @spyhunter99?

**Comment by spyhunter99** [comment]
Honestly, I was looking for a better replacement to pttdroid that wasn't gpl licensed. https://f-droid.org/repository/browse/?fdid=ro.ui.pttdroid

I'm afraid that this a bit out of my expertise. I already have enough open source projects that need attention.

**Comment by obscurerichard** [comment]
We had a good hackathon day at the end of Mil-OSS WG3. Maybe @Neutron can stir up some interest in reviving this; I think it could still be a valuable project.

**Comment by cpatrick** [comment]
Yeah, I don't have a ton of bandwidth atm, but it would be cool to hack on and modernize.

**Comment by Neutron** [comment]
The idea remains a good one .. and will become more pertinent once we start
using mobile devices more.  In the Everything Over IP world we are going to
need to have "radio" functionality without having to deal with all the
cross-band gateways etc. for converting between various proprietary
networking waveforms etc.

Currently the functionality is provided as functions of proprietary
stovepipe systems that not everyone will have .. so an app that does it
with any IP connection will be needed.

There are OSS Apps like Mumble which provide the basic functionality .. but
making it work like military tactical radios and implementing the military
use case is the challenge that remains.

I'm waiting for interest to peak in an appropriate project .. at which
point I will recommend the idea and the OSS collaborative development model
to ensure maximum availability and reduction of repetitive effort.

i don't have the cycles to work on it now either .. but when it comes up I
will try to instigate the right approach.

Neutron

**Comment by spyhunter99** [comment]
I'm pretty sure there's at least one government project that does this.
it's similiar in function and mechanics as PTT droid, but not open source,
or public for that matter
